platform_plugin_saleor/saleor_app/manifest.py

```diff
@@ -0,0 +1,112 @@
+"""Defines the manifest for the Saleor app."""
+
+from django.conf import settings
+
+
+def get_app_manifest():
+    """
+    Generate the application manifest for Saleor integration.
+
+    The manifest defines the application's metadata, permissions, webhooks,
+    and integration points that Saleor will use to interact with this plugin.
+
+    Returns:
+        dict: A dictionary containing the complete application manifest.
+    """
+
+    order_paid_query = """
```

Comment:
I think queries would be in a folder of querys or somethinkg like that in other module